config/l5-swagger.php

```diff
@@ -233,5 +233,6 @@
      */
     'constants' => [
         'L5_SWAGGER_CONST_HOST' => env('L5_SWAGGER_CONST_HOST', 'http://my-default-host.com'),
+        'L5_SWAGGER_CONST_GENERATE_OPERATION_ID_HASH' => env('L5_SWAGGER_CONST_GENERATE_OPERATION_ID_HASH', true),
```

Comment:
Why did you choose to put this as a const? Why not as a config key?

Reply:
Because constant "HOST" tells as about host for swagger instance. The flag for enabling hashing is also a parameter. I decided to do it by analogy with it.

Ideally, you need to create a separate section in the configuration file. I decided not to add it.

Reply:
The purpose of these constants in config was to replace constants (magic strings) in annotations with some value and generate documentation JSON with it. Example: https://github.com/DarkaOnLine/L5-Swagger/blob/master/tests/storage/annotations/OpenApi/Anotations.php#L20

Does this `L5_SWAGGER_CONST_GENERATE_OPERATION_ID_HASH ` constant fit here?